docs/advanced/chaos-experimentation.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Chaos Experimentation Framework
+{{ .EditURL }}
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+A framework to perform Chaos experiments built on top of [EnvoyProxy](https://www.envoyproxy.io/docs/envoy/latest/)
+
+<img alt="Clutch Component Architecture" src={useBaseUrl('img/docs/chaos-experimentation.png')} width="75%" />
+
+Chaos Experimentation Framework consists of few parts - frontend, a backend server and a xDS management server. 
+
+The Frontend uses [Clutch's core frontend](https://clutch.sh/docs/development/frontend). It can be customized by using the frontend config. 
+
+The Backend server is responsible for all the CRUD operations of the experiments. It stores experiments in its tables in the Postgres database. 
```

Comment:
is responsible for performing CRUD operations of the experimentaiton package or sth like this maybe. CRUD operations of the experiments sounds weird - CRUD operations related to experiments would be another way to say it. Maybe we could give some examples here - CreateExperiment? 
Maybe we could link to proto definition of these CRUD operations?

Reply:
I know that. you have some links later on but I think that a link to CRUD operations definition here would not hurt.